bundle/manifests/oadp-operator.clusterserviceversion.yaml

```diff
@@ -648,6 +648,8 @@ spec:
           - infrastructures
           verbs:
           - get
+          - list
+          - watch
```

Comment:
Why do you need list and watch? I'm only seeing get calls used here.

Reply:
yeah even I was unsure about those, I added because I was hitting list/watch permission errors in operator pod. I guess its harmless to add those.